patches/tModLoader/Terraria/TileObject.cs.patch

```diff
@@ -87,3 +95,17 @@
  				if (num36 != num38 || num37 != num39)
  					flag9 = true;
  			}
+@@ -743,7 +_,12 @@
+ 					if (op.Type == 114 && j == 1)
+ 						num12 += 2;
+ 
+-					sb.Draw(sourceRectangle: new Rectangle(x, num8, coordinateWidth, num12), texture: value, position: new Vector2(num9 * 16 - (int)(position.X + (float)(coordinateWidth - 16) / 2f) + drawXOffset, num10 * 16 - (int)position.Y + num5), color: color, rotation: 0f, origin: Vector2.Zero, scale: 1f, effects: spriteEffects, layerDepth: 0f);
```

Comment:
Please put this line a block comment instead of removing it. https://github.com/tModLoader/tModLoader/wiki/tModLoader-Style-Guide#comment-out-vanilla-code-rather-than-removing-it